### PR TITLE
add upload-image feature to addNewAttraction form.

### PR DESCRIPTION
**Pull request by camziny**
add image uploading feature to new attraction form. 

Co-authored-by: Alex Cameron <acameron724@users.noreply.github.com>
Co-authored-by: Fabricius Eusebio <yoFabricius@users.noreply.github.com>
Co-authored-by: JeanCGedeon <JeanCGedeon@users.noreply.github.com>

### CODE REVIEW

File: client/src/components/NewAttractionForm.js [hunk 1]
```diff
@@ -71,15 +81,33 @@ const NewAttractionForm = (props) => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
+    const body = new FormData();
+    body.append("image", newAttraction.image);
+    props.addNewAttraction(body);
```
Comment (dovakeen118) [inline client/src/components/NewAttractionForm.js]: I'm a little confused why there's both `props.addNewAttraction(body)` and a call on `postAttraction`? It also looks like `FormData` is created twice - above in this `handleSubmit`, and in the `postAttraction`. I don't see where the function `addNewAttraction` comes from?

File: server/src/config.js [hunk 2]
```diff
@@ -2,9 +2,23 @@ import "./boot.js";
 import getNodeEnv from "./config/getNodeEnv.js";
 import getDatabaseUrl from "./config/getDatabaseUrl.cjs";
 
-export default {
+const development = {
+  awsAccess: { key: process.env.AWS_ACCESS_KEY },
```
Comment (dovakeen118) [inline server/src/config.js:6]: We also need to update the `.env.example` file with placeholders for these new keys!

**Review by dovakeen118 — COMMENTED** [review]
Looking good! There's a question I left on the form about how the data is being posted to take a look at!

File: server/src/config.js (hunk 2 above)
Comment (camziny) [reply server/src/config.js:6]: .env example file has been updated

**Review by ohad-porat — APPROVED** [review]

**Pull request merged**